.travis.yml

```diff
@@ -1,8 +1,8 @@
-branches:
-  only:
-  - master
-  - develop
-  - /v\d.*/
+#branches:
+#  only:
+#  - master
+#  - develop
+#  - /v\d.*/
```

Comment:
Also: these should remain (we can adjust the branches if needed, but we shouldn't build every branch so that we can have flurry-building without bogging down PR builds)

Reply:
Done